### DISCUSSION TITLE
Desktop App (Tauri, Flutter)

### DISCUSSION THREAD

**Discussion by trymeouteh** [osmandapp/OsmAnd#20160] [Ideas]
Would like to see a downloadable desktop app for Windows, MacOS and Linux just like the mobile version.

I would suggest using Flutter or Tauri. Tauri allows you to create Windows, MacOS and Linux apps with one codebase. Flutter allows you to create apps for Windows, MacOS, Linux, Android and iOS with one codebase. Maybe it would be worth rewritting OSMAnd in Flutter to have OSMAnd as one codebase for mobile and desktop.

The OSMAnd website is cool but a desktop app would be incredible!

**Comment by dmpr0** [comment]
OsmAnd Maps (iOS version) is available for all Apple computers running Apple Silicon. We constantly working on Web version. Currently we don't have plans to make Flutter version.

**Reply by realgooseman** [reply]
> OsmAnd Maps (iOS version) is available for all Apple computers running Apple Silicon. We constantly working on Web version. Currently we don't have plans to make Flutter version.

Making a Flutter version isn't planned but could OSMAnd ever be ported to desktop ? Windows or Linux ?
If not, what could you recommend as a simple offline map downloader and viewer for Windows that works similarly to OSMAnd or Organic Maps ?

Tried QGIS and Grass GIS but are too complicated and OSM offline map usage isn't straightforward or not possible.

**Comment by mariush444** [comment]
OMC is written in jar so you can start it on any PC OS.
Then you can view what you expect as I guess.
What you viewd it is locally downloaded.
It is wotkaround but maybe enough for oyu.

**Reply by realgooseman** [reply]
> OMC is written in jar so you can start it on any PC OS. Then you can view what you expect as I guess. What you viewd it is locally downloaded. It is wotkaround but maybe enough for oyu.

Thanks, any links to the software and is it still maintained ?
I also found GPXSee in the meantime, where you can import map files.

Still haven't found a native windows app as good as OSMAnd or Organic Maps with offline capability and route planning.

**Reply by mariush444** [reply]
OMC is maintained continuously https://osmand.net/docs/versions/map-creator/
but at the 1st post you write about see map but at the last about "full" capability and rout planning.
So this is not the tool you are expected.

But maybe as a other workaround you can use https://osmand.net/docs/user/map/raster-maps/#download--update-tiles for downloading needed maps  area (raster tiles) to offline using.

here https://github.com/osmandapp/OsmAnd/discussions/11137 someone mantioned old offroad jar.
Maybe it will be useful for you.
Generally it is difficult to help because every one work in different way and finally find own method and solution.
Without knowing your exact needs and workflow it is just guessing.
